app/models/merchant.rb

```diff
@@ -60,4 +60,9 @@ def top_five_items
     .limit(5)
   end
 
+  def max_coupons
```

Comment:
Allows for error messages and removes activate button when false.  possible refactor for limit(5) and compare against it. or reset default status in table to be active.